editor/src/clj/editor/code/view.clj

```diff
@@ -1773,6 +1773,64 @@
                                                   (get-property view-node :cursor-ranges)
                                                   (get-property view-node :regions)))))
 
+(handler/defhandler :goto-definition :code-view
+  (enabled? [view-node]
+    (let [resource-node (get-property view-node :resource-node)
+          resource (g/node-value resource-node :resource)]
+      (resource/file-resource? resource)))
```

Comment:
We don't send open/change/close notifications for dependency files to the language servers because they can't open those files to keep up with editor changes. So when I ask for a definition at a particular cursor, the LSP server can't know what's the text there, because it can't load it from the disc, and it also can't scan the library file trees because it doesn't know about them.